.github/workflows/wasmedge-sys-docs.yml

```diff
@@ -0,0 +1,42 @@
+name: wasmedge-sys-docs
+
+concurrency:
+  group: rust-crate-release-${{ github.head_ref }}
+  cancel-in-progress: true
+
+on:
```

Comment:
Maybe we need to test the docs generation on master when the related rust files change. But we don't publish it until emitting the release tag.

Reply:
You're right. Using this strategy is based on the following consideration.  The current workflow is only triggered when there is a new tag coming, like `wasmedge-sys/0.4.0`. Such a setting can guarantee that the crate published on crates.io and the rustdoc published on the Github pages are built from the same commit. However, if we used `push-on-master` to trigger the workflow, the rustdoc would be built from the latest rust code, but not from the published version, that is, there would be version inconsistency between the published crate and the published rustdoc.